models.py

```diff
@@ -96,6 +96,7 @@ class AuditBoard(db.Model):
     member_1_affiliation = db.Column(db.String(200), nullable=True)
     member_2 = db.Column(db.String(200), nullable=True)
     member_2_affiliation = db.Column(db.String(200), nullable=True)
+    passphrase = db.Column(db.String(1000), nullable=True)
```

Comment:
Should there be a unique index on this?

Reply:
yeah, there probably should, and should also check for error conditions, will add those.